### DISCUSSION TITLE
Wiki BLACKARCH

### DISCUSSION THREAD

**Discussion by PardesiCat** [BlackArch/blackarch#3931] [General]
Type: Discussion Blackarch mirror

## Discussion
here in [blackarch-wiki ](url) mention rsync -zz   ( which is compression command ) 
but some rsync owners dont let sync using  compressed meathod   
 i will suggest  avoid -zz  meathod

![pcat](https://github.com/BlackArch/blackarch/assets/55957190/df656e41-ae7f-40aa-b103-b2a5fc5493b0)

2. there rsync -r  ( recursively)  missing this can cause problem 
i suggest add -r

**Comment by noraj** [comment]
Official website is https://blackarch.org/. https://blackarch.wiki/ is unofficial an run by @anunna

**Reply by anunna** [reply]
Thanks. 

I have added `-r` but will not exclude `-zz`. Excluding parameters simply because someone might not want it is up to the mirror host; it does not change the best practice recommendation. The cost of `-zz` is CPU only, 10-30% considering factors like CPU speed, number of cores, compression algorithm and other system loads. `-zz` is recommended for large mirrors, improving transfer rates and reducing bandwidth usage which produces a higher mirror availability rate.

Furthermore, the site is in development phase and not open for QA or contribution (hence private repo). Once complete and reviewed by the Team it will be opened for contribution if approved.
